plugins/outputs/redistimeseries/redistimeseries.go

```diff
@@ -0,0 +1,90 @@
+package redistimeseries
+
+import (
+	"fmt"
+
+	"github.com/go-redis/redis/v7"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+const sampleConfig = `
+  ## The address of the RedisTimeSeries server.
+  address = "127.0.0.1:6379"
+
+  ## Redis ACL credentials
+  # username = ""
+  # password = ""
+  # database = 0
+
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  # insecure_skip_verify = false
+`
+
+type RedisTimeSeries struct {
+	Address  string          `toml:"address"`
+	Username string          `toml:"username"`
+	Password string          `toml:"password"`
+	Database int             `toml:"database"`
+	Log      telegraf.Logger `toml:"-"`
+	tls.ClientConfig
+	client *redis.Client
+}
+
+func (r *RedisTimeSeries) Connect() error {
+	if r.Address == "" {
+		return fmt.Errorf("Redis address must be specified")
```

Comment:
Errors must start with lowercase according to Golang convention...
```suggestion
		return errors.New("redis address must be specified")
```